app/controllers/application_controller.rb

```diff
@@ -44,6 +48,18 @@ def infer_params_key(resource)
     klass.name.underscore.to_sym
   end
 
+# Sorting
+
+  # Sort by :id by default, so there is a consistent order even when no order has been
+  # selected via the interface
+  def default_sort_options
+    {id: :desc}
+  end
+
+  def sort(collection)
+    ModelSorter.sort(collection, params, default_sort_options)
```

Comment:
You can do that too.

We have pretty much 2 options:

1. keeping our controllers the same, and making the ApplicationController::sort infer the sort class
2. override sort method in children controllers

Both seems okay to me.